darwin/importer/importer.py

```diff
@@ -1680,3 +1696,53 @@ def _display_slot_warnings_and_errors(
             console.print(f"- File: {file}, issues:", style="info")
             for warning in slot_errors[file]:
                 console.print(f"  - {warning}")
+
+
+def _warn_for_annotations_with_multiple_instance_ids(
+    local_files: List[dt.AnnotationFile], console: Console
+) -> None:
+    """
+    Warns the user if any video annotations have multiple unique instance IDs.
+
+    This function checks each video annotation in the provided list of local annotation
+    files for multiple instance IDs. If any are found, a warning is printed to the console.
+    The user is then prompted to confirm if they want to proceed with the import.
+
+    Parameters
+    ----------
+    local_files : List[dt.AnnotationFile]
+        A list of local annotation files to be checked.
+    console : Console
+        The console object used to print warnings and messages.
+    """
+    files_with_multi_instance_id_annotations = {}
+    files_with_video_annotations = [
+        local_file for local_file in local_files if local_file.is_video
+    ]
+    for file in files_with_video_annotations:
+        for annotation in file.annotations:
+            unique_instance_ids = []
+            for frame_idx in annotation.frames:  # type: ignore
+                for subannotation in annotation.frames[frame_idx].subs:  # type: ignore
+                    if subannotation.annotation_type == "instance_id":
+                        instance_id = subannotation.data
+                        if instance_id not in unique_instance_ids:
+                            unique_instance_ids.append(instance_id)
+
+            if len(unique_instance_ids) > 1:
+                if file.path not in files_with_multi_instance_id_annotations:
+                    files_with_multi_instance_id_annotations[file.path] = 1
+                else:
+                    files_with_multi_instance_id_annotations[file.path] += 1
+
+    if files_with_multi_instance_id_annotations:
+        console.print(
+            "The following files have at least one annotation with multiple instance IDs. Your team has static instance IDs enabled, so only the first instance ID of each annotation will be imported:",
+            style="warning",
+        )
+        for file in files_with_multi_instance_id_annotations:
+            console.print(
+                f"- File: {file} has {files_with_multi_instance_id_annotations[file]} annotation(s) with multiple instance IDs"
+            )
+        if not secure_continue_request():
+            raise SystemExit("Execution stopped by user.")
```

Comment:
I'm not sure if we want to throw a warning that requires input to continue here. If we do want to, maybe we should provide a way of bypassing the warning? Open to feedback